src/app/pages/details/common/resource-box/resource-box.js

```diff
@@ -45,7 +45,7 @@ function resourceBoxPermissions({
 // is available to the user (instructor version)
 export function instructorResourceBoxPermissions(resourceData, userStatus, search) {
     const resourceStatus = () => {
-        if (resourceData.resourceUnlocked || userStatus.isInstructor) {
+        if (resourceData?.resource?.resourceUnlocked || userStatus.isInstructor) {
```

Comment:
I'm super unfamiliar with os-web, where does this nested resource come from? Is this related to the CTA change?

Reply:
No, it's a tacked-on fix for something Ed pointed out the other day. I'll put it in a separate PR.
![image](https://github.com/openstax/os-webview/assets/7315241/00b42a79-d3dc-4201-ae3c-e707ee90999a)

Reply:
Ah thanks for the clarification. I'll probably have lots of questions as I get to know the codebase :)